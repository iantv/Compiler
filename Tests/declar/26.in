struct a{
	int c;
};
void f(struct a b);