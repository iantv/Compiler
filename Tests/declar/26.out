f: function
	b: parameter
		struct a
			c: variable
				int
returns
	void
struct a
	c: variable
		int
